src/test/java/org/jabref/logic/importer/fetcher/SpringerFetcherTest.java

```diff
@@ -80,6 +80,12 @@ void searchByEmptyQueryFindsNothing() throws Exception {
         assertEquals(Collections.emptyList(), fetcher.performSearch(""));
     }
 
+    @Test
+    @Disabled("Year search is currently broken, because the API returns mutliple years.")
```

Comment:
I'm a bit split between adding disabled here or not. The fetcher doesn't adhere to the interface tested here, so it's correct that the test is failing, or not? I'm a bit afraid that once you mark tests as disabled, no one will try to fix it because well there are no failing tests, right?

Reply:
Added to the agenda of JabCon